test/SignalRServiceExtension.Tests/Trigger/SignalRMethodExecutorTests.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR.Protocol;
+using Microsoft.Azure.SignalR.Serverless.Protocols;
+using Microsoft.Azure.WebJobs.Extensions.SignalRService;
+using Microsoft.Azure.WebJobs.Extensions.SignalRService.Tests.Common;
+using Microsoft.Azure.WebJobs.Host.Executors;
+using Moq;
+using Newtonsoft.Json;
+using SignalRServiceExtension.Tests.Utils;
+using Xunit;
+using ExecutionContext = Microsoft.Azure.WebJobs.Extensions.SignalRService.ExecutionContext;
+
+namespace SignalRServiceExtension.Tests.Trigger
+{
+    public class SignalRMethodExecutorTests
```

Comment:
>SignalRMethodExecutorTests [](start = 17, length = 26)

Add some tests to cover IRequestResolver return validation false as well?

Reply:
I added some, and put it in the DispatcherTest